### PR TITLE
fix(global): pf-global and pf-v5 to pf-v6 updates

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/6493

### CODE REVIEW

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 1]
```diff
@@ -38,7 +38,7 @@
   }
 
   // Close
-  --#{$about-modal-box}__close--ZIndex: var(--#{$pf-global}--ZIndex--2xl); // TODO zindex token
+  --#{$about-modal-box}__close--ZIndex: var(--pf-t--global--Zindex--2xl); // TODO zindex token
```
Comment (srambach) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss:41]: Can probably take the TODO off now?

File: src/patternfly/components/Content/content.scss [hunk 2]
```diff
@@ -86,7 +86,7 @@
   --#{$content}--ul--MarginLeft: var(--pf-t--global--spacer--lg);
   --#{$content}--ul--nested--MarginTop: var(--pf-t--global--spacer--sm);
   --#{$content}--ul--nested--MarginLeft: var(--pf-t--global--spacer--sm);
-  --#{$content}--ul--ListStyle: var(--#{$pf-global}--ListStyle); // TODO
+  --#{$content}--ul--ListStyle: var(--pf-t--global--list-style); // TODO
```
Comment (srambach) [inline src/patternfly/components/Content/content.scss:89]: Does this still need a TODO?

File: src/patternfly/components/DualListSelector/dual-list-selector.scss [hunk 3]
```diff
@@ -82,7 +82,7 @@ $pf-v5-c-dual-list-selector__item--MaxNesting: 10;
   // Icon
   --#{$dual-list-selector}__item-toggle-icon--Rotate: 0;
   --#{$dual-list-selector}__list-item--m-expanded__item-toggle-icon--Rotate: 90deg;
-  --#{$dual-list-selector}__item-toggle-icon--Transition: var(--#{$pf-global}--Transition); // need token
+  --#{$dual-list-selector}__item-toggle-icon--Transition: var(--pf-t--global--transition); // need token
```
Comment (srambach) [inline src/patternfly/components/DualListSelector/dual-list-selector.scss:85]: I think this comment can be removed

File: src/patternfly/components/JumpLinks/jump-links.scss [hunk 4]
```diff
@@ -65,7 +65,7 @@ $pf-v6-c-jump-links--m-expandable--breakpoint-map: build-breakpoint-map("base",
 
   // toggle icon
   --#{$jump-links}__toggle-icon--Color: currentcolor;
-  --#{$jump-links}__toggle-icon--Transition: var(--#{$pf-global}--Transition); // TODO
+  --#{$jump-links}__toggle-icon--Transition: var(--pf-t--global--transition); // TODO
```
Comment (srambach) [inline src/patternfly/components/JumpLinks/jump-links.scss:68]: Another TODO that's DONE :-)

File: src/patternfly/components/List/list.scss [hunk 5]
```diff
@@ -5,7 +5,7 @@
   --#{$list}--PaddingLeft: var(--pf-t--global--spacer--lg);
   --#{$list}--nested--MarginTop: var(--pf-t--global--spacer--sm);
   --#{$list}--nested--MarginLeft: var(--pf-t--global--spacer--sm);
-  --#{$list}--ul--ListStyle: var(--#{$pf-global}--ListStyle); // TODO add token?
+  --#{$list}--ul--ListStyle: var(--pf-t--global--list-style); // TODO add token?
```
Comment (srambach) [inline src/patternfly/components/List/list.scss:8]: DONE 🎃

File: src/patternfly/components/Menu/menu.scss [hunk 6]
```diff
@@ -121,17 +121,17 @@
   --#{$menu}--m-flyout__menu--m-left--Right: calc(100% + var(--#{$menu}--m-flyout__menu--m-left--right-offset));
 
   // * Menu drilldown content
-  --#{$menu}--m-drilldown__content--TransitionDuration--height: var(--#{$pf-global}--TransitionDuration); // TODO: Update transition with token update
-  --#{$menu}--m-drilldown__content--TransitionDuration--transform: var(--#{$pf-global}--TransitionDuration); // TODO: Update transition with token update
+  --#{$menu}--m-drilldown__content--TransitionDuration--height: var(--pf-t--global--transitionDuration); // TODO: Update transition with token update
+  --#{$menu}--m-drilldown__content--TransitionDuration--transform: var(--pf-t--global--transitionDuration); // TODO: Update transition with token update
```
Comment (srambach) [inline src/patternfly/components/Menu/menu.scss:125]: ✅  Can remove these TODOs

File: src/patternfly/components/Table/table.scss [hunk 7]
```diff
@@ -115,7 +115,7 @@
   --#{$table}__action--PaddingRight: var(--pf-t--global--spacer--sm);
 
   // * Table expandable row
-  --#{$table}__expandable-row--Transition: var(--#{$pf-global}--Transition); // TODO: update to token when available
+  --#{$table}__expandable-row--Transition: var(--pf-t--global--transition); // TODO: update to token when available
```
Comment (srambach) [inline src/patternfly/components/Table/table.scss:118]: TODONE

Comment (srambach) [inline src/patternfly/components/Table/table.scss:118]: TODONE

File: src/patternfly/base/_themes.scss [hunk 8]
```diff
@@ -6,38 +6,38 @@
   }
 
   &.pf-m-transparent-100 {
-    background-color: rgba($pf-v5-color-black-1000, .42);
+    background-color: rgba($pf-v6-color-black-1000, .42);
```
Comment (srambach) [inline src/patternfly/base/_themes.scss:9]: Do we even still need this file? Is it used for those deprecated components still?

File: src/patternfly/components/Menu/menu.scss (hunk 6 above)
Comment (srambach) [inline src/patternfly/components/Menu/menu.scss:125]: These TODOs can go!

File: src/patternfly/components/Menu/menu.scss [hunk 9]
```diff
@@ -121,17 +121,17 @@
...
 
   // * Menu drilldown menu
   --#{$menu}--m-drilldown--c-menu--Top: 0;
-  --#{$menu}--m-drilldown--c-menu--TransitionDuration--transform: var(--#{$pf-global}--TransitionDuration);
+  --#{$menu}--m-drilldown--c-menu--TransitionDuration--transform: var(--pf-t--global--transitionDuration);
   --#{$menu}--m-drilldown--c-menu--Transition: transform var(--#{$menu}--m-drilldown--c-menu--TransitionDuration--transform);
 
   // * Menu drilldown list
-  --#{$menu}--m-drilldown__list--TransitionDuration--transform: var(--#{$pf-global}--TransitionDuration); // TODO: Update transition with token update
+  --#{$menu}--m-drilldown__list--TransitionDuration--transform: var(--pf-t--global--transitionDuration); // TODO: Update transition with token update
```
Comment (srambach) [inline src/patternfly/components/Menu/menu.scss:134]: TODO: remove TODO

File: src/patternfly/components/ModalBox/modal-box.scss [hunk 10]
```diff
@@ -4,7 +4,7 @@
   --#{$modal-box}--BackgroundColor: var(--pf-t--global--background--color--floating--default);
   --#{$modal-box}--BorderRadius: var(--pf-t--global--border--radius--large);
   --#{$modal-box}--BoxShadow: var(--pf-t--global--box-shadow--lg);
-  --#{$modal-box}--ZIndex: var(--#{$pf-global}--ZIndex--xl); // TODO use a z-index token
+  --#{$modal-box}--ZIndex: var(--pf-t--global--Zindex--xl); // TODO use a z-index token
```
Comment (srambach) [inline src/patternfly/components/ModalBox/modal-box.scss:7]: ✅ Remove TODO

File: src/patternfly/components/Sidebar/sidebar.scss [hunk 11]
```diff
@@ -74,7 +74,7 @@ $pf-v6-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
 
   // Panel
   --#{$sidebar}__panel--FlexBasis--base: auto;
-  --#{$sidebar}__panel--BoxShadow--base: #{0 pf-size-prem(4px) pf-size-prem(4px) pf-size-prem(-4px) rgba($pf-v5-color-black-1000, .16)}; // TODO NEEDS TOKEN. previous comment:modified so the shadow is only visible on the bottom of the element 
+  --#{$sidebar}__panel--BoxShadow--base: var(--pf-t--global--box-shadow--md--bottom); // TODO NEEDS TOKEN. previous comment:modified so the shadow is only visible on the bottom of the element
```
Comment (srambach) [inline src/patternfly/components/Sidebar/sidebar.scss:77]: This comment is resolved

File: src/patternfly/components/Toolbar/toolbar.scss [hunk 12]
```diff
@@ -52,7 +52,7 @@ $pf-v6--include-toolbar-breakpoints: true !default;
 
   // * Toolbar expand all
   --#{$toolbar}__expand-all-icon--Rotate: 0;
-  --#{$toolbar}__expand-all-icon--Transition: var(--#{$pf-global}--Transition); // TODO: update when transition is tokenized
+  --#{$toolbar}__expand-all-icon--Transition: var(--pf-t--global--transition); // TODO: update when transition is tokenized
```
Comment (srambach) [inline src/patternfly/components/Toolbar/toolbar.scss:55]: All done! TODO can be removed.

**Review by srambach — APPROVED** [review]
Just silly comments about removing TODO markers.
FABULOUS job! 🐰

File: src/patternfly/components/Table/table.scss (hunk 7 above)
Comment (mcoker) [reply src/patternfly/components/Table/table.scss:118]: That would be a great productivity app name!

File: src/patternfly/base/_themes.scss (hunk 8 above)
Comment (mcoker) [reply src/patternfly/base/_themes.scss:9]: NOPE!

**Review by srambach — APPROVED** [review]
Perfecto! 👌

**Pull request merged**